<?xml version="1.0"?>

<launch>

  <!--<include file="$(find can_to_eth)/launch/c2e_pro.launch">
    <arg name="host" default="192.168.2.7" />
  </include>-->

  <!--<env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find mypackage)/custom_rosconsole.conf"/>-->

  <!--Run tf publisher-->
  <node name="tf_publisher" pkg="decision" type="tf_publisher" />

  <!-- Run the map server -->
  <!--<node name="rviz" pkg="rviz" type="rviz" args="-d $(find decision)/urdf/navigation.rviz"/>-->
  <node name="decision_pub" pkg="decision" type="decision_pub" output="screen"/>
    <!-- dynamic reconfigure-->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>

</launch>